### DISCUSSION TITLE
Graylog warning.

### DISCUSSION THREAD

**Discussion by fordcrews** [RobertDWhite/WhiteMatterTech#15] [General]
I get this warning Elasticsearch nodes with too low open file limit (triggered 2 hours ago)
There are Elasticsearch nodes in the cluster that have a too low open file limit (current limit: 40,960 on 172.21.0.3; should be at least 64000) This will be causing problems that can be hard to diagnose. Read how to raise the maximum number of open files in [the Elasticsearch setup documentation](https://docs.graylog.org/docs/elasticsearch#configuration).

I have tried to increase the ulimit, and must have done something wrong, got an error about not being able to run ulimit, then when I took it out of the docker startup script, it wouldn't start until I deleted the entire thing and started over.   The warning doesn't seem to be effecting anything on my small setup, but hate seeing warnings.

**Comment by RobertDWhite** [comment]
@fordcrews I had the same errors in my configuration as well. I have since removed Unraid from my lab for now. However, you might try adding/updating the following lines to elasticsearch in the docker-compose file:

```
    ulimits:
      memlock:
        hard: -1
        soft: -1
      nofile:
        hard: 65536
        soft: 65536 
```

Let us know if that fixes for you or not.
Thanks!
